api/src/modules/projects/projects.controller.ts

```diff
@@ -48,12 +44,11 @@ export class ProjectsController {
   }
 
   @TsRestHandler(projectsContract.getProjectsMap)
-  async getProjectsMap(
-    @ProcessFetchSpecification() dto: FetchSpecification,
-  ): ControllerResponse {
-    return tsRestHandler(projectsContract.getProjectsMap, async () => {
+  async getProjectsMap(): ControllerResponse {
+    return tsRestHandler(projectsContract.getProjectsMap, async ({ query }) => {
```

Comment:
this was the bug for countryCodes, where if we send >1 values, it would work, but not with a single value since there is a mismatch between what the client sends and what the api expects due to the contract and lack of validation